sdk/tests/wallet/claim_outputs.rs

```diff
@@ -75,15 +75,31 @@ async fn claim_2_basic_outputs_no_outputs_in_claim_account() -> Result<()> {
 
     // Equal to minimum required storage deposit for a basic output
     let micro_amount = 42600;
-    let tx = account_0
-        .send_amount(
-            vec![
-                AddressWithAmount::new(account_1.addresses().await?[0].address().to_bech32(), micro_amount),
-                AddressWithAmount::new(account_1.addresses().await?[0].address().to_bech32(), micro_amount),
-            ],
-            None,
-        )
-        .await?;
+
+    let token_supply = account_0.client().get_token_supply().await?;
+    let rent_structure = account_0.client().get_rent_structure().await?;
+    let expiration_time = account_0.client().get_time_checked().await? + 86400; // 1 Day from now
+
+    // Get the output's rent cost
+    let output = BasicOutputBuilder::new_with_amount(micro_amount)?
+        .add_unlock_condition(AddressUnlockCondition::new(
+            *account_1.addresses().await?[0].address().as_ref(),
+        ))
+        .add_unlock_condition(ExpirationUnlockCondition::new(
+            *account_0.addresses().await?[0].address().as_ref(),
+            expiration_time,
+        )?)
```

Comment:
Sorry what I sent missed the SDR, I think it should be
```suggestion
        )?)
        .add_unlock_condition(StorageDepositReturnUnlockCondition::new(
              *account_0.addresses().await?[0].address().as_ref(),
              micro_amount,
              token_supply,
        )?)
```

Reply:
Ok actually not needed since we don't need to send something back